src/nvim/popupmnu.c

```diff
@@ -219,7 +220,9 @@ void pum_display(pumitem_T *array, int size, int selected)
     if (curwin->w_p_rl) {
       pum_width = pum_col - pum_scrollbar + 1;
     } else {
-      pum_width = Columns - pum_col - pum_scrollbar;
+      assert(Columns - pum_col - pum_scrollbar >= INT_MIN
```

Comment:
You're right, but I'm not sure whether to remove it or not.
In cases where it's not evident I think I would leave them.
Also, this kind of assert should be eventually replaces by `IN_BOUNDS(<type>)` macro, which won't have the change to make such optimizations. So, we gain nothing by doing them now. Do you agree?